code/modules/shuttle/on_move.dm

```diff
@@ -16,13 +16,13 @@
     // Stationary ports shouldn't move, mobile ones move themselves
     return 0
 
-/obj/machinery/door/onShuttleMove()
+/obj/machinery/door/airlock/onShuttleMove()
```

Comment:
@tigercat2000 you think this should happen to all doors, but exclude firedoors or only airlocks?